File: cpe-level-1/python/zj-c067.py
#!/usr/bin/env python 

iters = 0

while True:
    n = int(input())
    if  n == 0:
        break
    bricks = list(map(int, input().split()))
    avg = sum(bricks) // n
    steps = 0
    for b in bricks:
        if b > avg:
            steps += b - avg
    iters += 1
    print(f"Set #{iters}")
    print(f"The minimum number of moves is {steps}.")

File: cpe-level-1/cpp/zj-c067.cpp

#include <iostream>
using namespace std;

int main() {
    int n, T = 1;
     
    while (cin >> n && n) {
        int sum = 0;
        int a[n];
        for (int i = 0; i < n; i++) {
            cin >> a[i];
            sum += a[i];
        }
        int avg = sum / n;
        int ans = 0;
        for (int i = 0; i < n; i++) {
            if (a[i] > avg)
                ans += a[i] - avg;
        }
        cout << "Set #" << T++ << "\n";
        cout << "The minimum number of moves is " << ans << ".\n\n";
    }
    return 0;
}
